### ISSUE TITLE
Payment pointer Bob fails

### ISSUE THREAD

**Issue by bourgeoa** [nodeSolidServer/node-solid-server#1717]
@michielbdejong 
I skipped the corresponding test :
```
 1) API
       Payment Pointer Bob
         GET Payment Pointer document
           should redirect to example.com:
     Error: write EPROTO 139799603906496:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:

      at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:94:16)
```

**Comment by michielbdejong** [comment]
Thanks for the heads-up.

Ah, this started failing because the cert on https://bob.com/ expired!
Should be easy to fix by changing "bob.com" to "example.com", or some other domain name.

**Comment by bourgeoa** [comment]
replacing bob.com with example.com or service.com did not resolve the issue.
https://github.com/nodeSolidServer/node-solid-server/blob/0bb5e9924b11656a9e4a51e5035d988455b04576/test/integration/payment-pointer-test.js#L108
```
Error: write EPROTO 140539925039040:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:
```

**Comment by zg009** [comment]
@bourgeoa This test is passing for me on main branch when I removed the skip. I have self-signed certificates that I created less than a month ago. That could be your issue?

**Comment by bourgeoa** [comment]
@zg009 No explanation. My keys are young and the test is still failing
```
# ls -al test/keys
total 32
drwxrwxrwx 1 root root  512 Mar 10 10:54 .
drwxrwxrwx 1 root root  512 Mar 23 14:55 ..
-rwxrwxrwx 1 root root 1285 Mar 10 10:54 cert.pem
-rwxrwxrwx 1 root root 1107 Mar 10 10:54 client-cert.pem
-rwxrwxrwx 1 root root 1679 Mar 10 10:54 client-key.pem
-rwxrwxrwx 1 root root 1679 Mar 10 10:54 key.pem
-rwxrwxrwx 1 root root 1386 Mar 10 10:54 user1-cert.pem
-rwxrwxrwx 1 root root 1703 Mar 10 10:54 user1-key.pem
-rwxrwxrwx 1 root root 1314 Mar 10 10:54 user2-cert.pem
-rwxrwxrwx 1 root root 1675 Mar 10 10:54 user2-key.pem
```

**Comment by zg009** [comment]
@bourgeoa I am running NSS tests on WSL v2 in Ubuntu 20.04. My version of OpenSSL is 1.1.1f. It may be a versioning/OS issue.

**Comment by bourgeoa** [comment]
@zg009 
You are right the test passes with WSL2 in ubuntu 20.04, but fails on WSL 1 that I am normally using
On the contrary with WSL2 an other test fails in http-test.js (POST multipart) but succeed in WSL1

Strange

**Comment by zg009** [comment]
> @zg009 You are right the test passes with WSL2 in ubuntu 20.04, but fails on WSL 1 that I am normally using On the contrary with WSL2 an other test fails in http-test.js (POST multipart) but succeed in WSL1
> 
> Strange

Can you point out the line it http-test POST which fails? All of mine are passing on WSL2.

**Comment by bourgeoa** [comment]
This is the test that fails for me on WSL2
https://github.com/nodeSolidServer/node-solid-server/blob/21096e209d7c6e23f67c8b4bf9ad546301ebf4a3/test/integration/http-test.js#L1161

**Comment by zg009** [comment]
> This is the test that fails for me on WSL2
> 
> https://github.com/nodeSolidServer/node-solid-server/blob/21096e209d7c6e23f67c8b4bf9ad546301ebf4a3/test/integration/http-test.js#L1161

This is working for me on WSL2 on the main branch. I'm not sure what the issue could be just scanning the file. Can you rerun it and show the full error? I'm wondering where it is failing the test.
